lib/index.ts

```diff
@@ -10,6 +10,7 @@ export * from './navigation/index.ts'
 export * from './newMenu/index.ts'
 export * from './node/index.ts'
 export * from './permissions.ts'
+export * from './types.ts'
```

Comment:
Maybe safe guard this 


```suggestion
export type * from './types.ts'
```